src/shell/autoexec.cpp

```diff
@@ -0,0 +1,584 @@
+/*
+ *  Copyright (C) 2020-2023  The DOSBox Staging Team
```

Comment:
`list` is (almost!) shadowing `std::list`.

Does `list` hold `autoexec_lines` (is that better name, or maybe just `lines`?

Reply:
Ooops - looks like this comment was not against the right lines.

It's in the `for( const auto& [source, list] :  ..`) loop.

Reply:
It is now `for (const auto& [source, list_lines] : autoexec_lines) {` - so no longer close to shadowing anything.